### ISSUE TITLE
Error: no overload matches 'Awscr::S3::Part.new', offset: (Float64 | Int32)...

### ISSUE THREAD

**Issue by joanrega** [taylorfinnell/awscr-s3#62]
Hi!
I'm using your project to upload files to S3, and I've had no problem thus far, and it is actually working, but when spec tests are thrown at it, I get this error.
```

In lib/awscr-s3/src/awscr-s3/multipart_file_uploader.cr:46:26

 46 | @pending << Part.new(
                       ^--
Error: no overload matches 'Awscr::S3::Part.new', offset: (Float64 | Int32), size: Int32, number: Int32

Overloads are:
 - Awscr::S3::Part.new(offset : Int32, size : Int32, number : Int32)
Couldn't find overloads for these types:
 - Awscr::S3::Part.new(offset : Float64, size : Int32, number : Int32)
```

So I've found that apparently the `compute_default_part_size` function returns the Float64. If I convert the output to an Int32 at line 42 it passes.

I'm sorry for the problems, it's just that to make sure it consistently works I want to check this.

**Comment by taylorfinnell** [comment]
Hi, what version of the shard and Crystal are you using? I think this was fixed in #57

**Comment by joanrega** [comment]
Okay, let me sink in shame here.

> Hi, what version of the shard and Crystal are you using? I think this was fixed in #57

Okay, sorry to bother you, it has been a docker configuration problem, getting 0.31 and using 0.5 of your library. I was convinced they set up to a compatible one. Shame on me for not checking that one before.
Thank you for your attention and for the library!
